protocols/README.md

```diff
@@ -0,0 +1,8 @@
+# Documentation/Protocols
+
+## Known `make` errors
+
+| Symptom | Fix | Notes |
+| ------- | --- | ----- |
+| `groff: couldn't exec refer: No such file or directory` | `$ apt install groff`, or equivalent package manager command. | *As per issue #11* |
+| `troff: fatal error: can't find macro file s` | `$ apt install groff`, or equivalent package manager command. | *As per issue #11* |
```

Comment:
Maybe we should just mark `groff` as a required install in order to properly build the documentation. Which is true, in all the cases you describe in your table